### PR TITLE
Fixed issue where track resumed instead of playing the next.

### PR DESCRIPTION
**Pull request by fela98**
This solves #103 .

### CODE REVIEW

File: MediaManager/Plugin.MediaManager.iOS/AudioPlayerImplementation.cs [hunk 1]
```diff
@@ -218,7 +218,10 @@ private void InitializePlayer()
             if(mediaFile != null)
                 nsUrl = new NSUrl(mediaFile.Url);
 
-            if (Status == MediaPlayerStatus.Paused)
+            var currentPlayerAsset = (AVUrlAsset) Player.CurrentItem?.Asset;
+            var currentPlayerItemUrl = currentPlayerAsset?.Url?.ToString();
+
+            if (Status == MediaPlayerStatus.Paused && mediaFile.Url == currentPlayerItemUrl)
```
Comment (martijn00) [inline MediaManager/Plugin.MediaManager.iOS/AudioPlayerImplementation.cs]: Are you sure that comparing the url is a reliable way to check this?

Comment (fela98) [inline MediaManager/Plugin.MediaManager.iOS/AudioPlayerImplementation.cs]: The only problem I could imagine is if you have different mediaFiles with the same Url, which in my opinion should never happen. What else would you do here? Save the instance in a private field and compare it later? Or is there something else we can use for identification?

Comment (fela98) [inline MediaManager/Plugin.MediaManager.iOS/AudioPlayerImplementation.cs]: I think the bigger question here is what identifies a MediaFile uniquely and how we should compare them.

**Comment by modplug** [comment]
Once we decide we should implement IEquatable on MediaFile and use that for
comparison

tor. 26. jan. 2017 kl. 08.14 skrev Felix Schoeler <notifications@github.com

**Comment by fela98** [comment]
Replaced implementation with one using IEquatable.

**Pull request merged**